.circleci/lint_pr.sh

```diff
@@ -8,5 +8,5 @@ if [ -z "$PR_NUM" ]; then
 fi
 
 # get PR file list, filter out removed files, filter only JS/TS files, then pass to the linter
-curl -fsSL https://api.github.com/repos/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PR_REPONAME/pulls/$PR_NUM/files | jq -r '.[] | select(.status!="removed") | .filename' | grep -E '\.(js|jsx|ts|tsx)$' || true | xargs yarn eslint
```

Comment:
`|| true` was causing bash to skip rest of command, so this was just printing file list ....

Before
![image](https://user-images.githubusercontent.com/5849952/217653668-0da3695f-5b37-498d-9ecb-cb8313451b63.png)


After